.eslintrc.cjs

```diff
@@ -0,0 +1,5 @@
+const eslint = require("@discourse/lint-configs/eslint-theme");
+
+const config = { ...eslint };
+
+module.exports = config;
```

Comment:
If there are no changes to the stock config, you can re-export it like the other two configs:

```suggestion
module.exports = require("@discourse/lint-configs/eslint-theme");
```